packages/percept/CMakeLists.txt

```diff
@@ -208,7 +208,9 @@ IF(TARGET perceptX)
     TARGET percept perceptX
     PROPERTY COMPILE_DEFINITIONS STK_PERCEPT_LITE=0 STK_PERCEPT_USE_INTREPID
   )
-  install_executable(perceptX)
+#  install_executable(perceptX)
+  INSTALL(TARGETS ${target} COMPONENT ${PACKAGE_NAME} RUNTIME DESTINATION ${${PROJECT_NAME}_INSTALL_RUNTIME_DIR})
```

Comment:
I'd like to get rid of the code wholesale instead of commenting it out.

I take it that the issue was a missing `percept` install?